src/stylesheets/components/_footer.scss

```diff
@@ -1,130 +1,30 @@
 //
 // Footer Component
 //
-// Based on the existing GOV.UK footer in GOV.UK Template
+// GOV.UK Frontend footer adapted
 
 @include govuk-exports("app-footer") {
-  $app-footer-background: $govuk-grey-3;
-  $app-footer-border-top: #a1acb2;
-  $app-footer-link: #454a4c;
-  $app-footer-text: $app-footer-link;
-  $app-footer-link-hover: #171819;
-
-  .app-c-footer {
-    display: block;
-    padding: $govuk-gutter $govuk-spacing-scale-3 ($govuk-gutter * 2);
-    border-top: 1px solid $app-footer-border-top;
-    color: $app-footer-text;
-    background: $app-footer-background;
+  .govuk-c-footer--full .govuk-o-width-container {
```

Comment:
1. We should try and avoid using `govuk-` here instead using `app-`
2. You can use `containerClass` to target directly where the `.govuk-o-width-container` is applied.

Reply:
oh, missed that param, ta